tests/model_find_all_group_test.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+require('./helper');
+
+const { expect } = require('chai');
+const { Sequelize, DataTypes } = require('../source');
+
+describe('Model', () => {
+  describe('findAll', () => {
+    describe('group', () => {
+      it('should correctly group with attributes, #3009', async function() {
+        const Post = this.sequelize.define('Post', {
+          id: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
+          name: { type: DataTypes.STRING, allowNull: false }
+        });
+
+        const Comment = this.sequelize.define('Comment', {
+          id: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
+          text: { type: DataTypes.STRING, allowNull: false }
+        });
+
+        Post.hasMany(Comment);
+
+        await this.sequelize.sync({ force: true });
+
+        // CRDB does not give ids 1 and 2. It is usually a Big Number.
```

Comment:
I would update this comment to be a little more specific. Note that autoIncrement does not guarantee the integers to start at 1.